components-core/src/main/java/com/adyen/checkout/components/core/internal/analytics/CheckoutAttemptIdState.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 8/10/2024.
+ */
+
+package com.adyen.checkout.components.core.internal.analytics
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed class CheckoutAttemptIdState {
+    data class Available(val checkoutAttemptId: String) : CheckoutAttemptIdState()
```

Comment:
If you put `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` on all these sub classes, then they shouldn't show up in the .api file